dist/linux/info.cemu.Cemu.metainfo.xml

```diff
@@ -24,7 +24,7 @@
     <p xml:lang="el">Το Cemu είναι ένας προσομοιωτής της κονσόλας Nintendo Wii U που υποστηρίζει τεπίσημα παιχνίδια, καθώς και μη επίσημα προγράμματα ("homebrew"), για το Wii U. Δημιουργήθηκε από τον Exzap σε C/C++.</p>
     <p xml:lang="es">Cemu es un emulador de Nintendo Wii U que es capaz de ejecutar la mayoría de los juegos de Wii U y homebrew en un estado jugable. Creado por Exzap, y escrito en C y C++.</p>
     <p xml:lang="pt_BR">Cemu é um emulador de Nintendo Wii U que é capaz de executar a maioría dos jogos de Wii U e homebrew em um estado jogavel. Criado por Exzap, e escrito em C e C++.</p>
-    <p xml:lang="it">Cemu è un emulatore del Nintendo Wii U capace di riprodurre la maggior parte dei giochi ed homebrew Wii U in stato giocabile. Creato da Exzap, e scritto in C/C++.</p>
+    <p xml:lang="it">Cemu è un emulatore del Nintendo Wii U in grado di eseguire la maggior parte dei giochi e homebrew per Wii U in stato giocabile. Creato da Exzap e scritto in C/C++.</p>
```

Comment:
I wouldn't remove the comma in `Creato da Exzap,` as the original string has it too.

Reply:
The comma sounded a bit "forced" to me while reading the sentence, but in the end it's all a matter of style [as the Crusca says](https://accademiadellacrusca.it/it/consulenza/uso-della-virgola-prima-della-congiunzione-e/148).
I don't have a strong opinion on this btw, I'm fine with reverting it too.

Reply:
I agree, but I prefer translations to be faithful to the original if possible. As you say, it's just a matter of style :)

Reply:
Reverted. I've also restructured the sentence a bit and added the `GenericName` to the .desktop file.